TM1py/Services/GitService.py

```diff
@@ -0,0 +1,275 @@
+        :param username: GIT username
+        :param password: GIT password
+        :param public_key: SSH public key, available from PAA V2.0.9.4
+        :param private_key: SSH private key, available from PAA V2.0.9.4
+        :param passphrase: Passphrase for decrypting private key, if set
+        :param force: reset git context on True
+        :param config: Dictionary containing git configuration parameters
+        """
+        url = "/api/v1/GitInit"
+        body = {}
+        body['URL'] = git_url
+        body['Deployment'] = deployment
+
+        if username is not None:
+            body['Username'] = username
+        if password is not None:
+            body['Password'] = password
+        if public_key is not None:
+            body['PublicKey'] = public_key
+        if private_key is not None:
+            body['PrivateKey'] = private_key
+        if passphrase is not None:
+            body['Passphrase'] = passphrase
+        if force is not None:
+            body['Force'] = force
+        if config is not None:
+            body['Config'] = config
+        
+        for key, value in kwargs.items():
+            body[key] = value
+
+        body_json = json.dumps(body)
+        response = self._rest.POST(url=url, data=body_json)
+        return self.from_json(response.json())
+
+    def git_uninit(self, force=False):
+        """ Unitialize GIT service
+        :param Force: clean up gitcontext when True
+        """
+        url = "/api/v1/GitUninit"
+        body = json.dumps(force)
+        return self._rest.POST(url=url, data=body)
+
+    def git_status(self, username: str = None, password: str = None, public_key: str = None, private_key: str = None,
+                    passphrase: str = None, **kwargs) -> Git:
+        """ Get GIT status, returns Git object
+        :param Username: GIT username
+        :param Password: GIT password
+        :param PublicKey: SSH public key, available from PAA V2.0.9.4
+        :param PrivateKey: SSH private key, available from PAA V2.0.9.4
+        :param Passphrase: Passphrase for decrypting private key, if set
+        """
+        url = "/api/v1/GitStatus"
+        body = {}
+
+        if username is not None:
+            body['Username'] = username
+        if password is not None:
+            body['Password'] = password
+        if public_key is not None:
+            body['PublicKey'] = public_key
+        if private_key is not None:
+            body['PrivateKey'] = private_key
+        if passphrase is not None:
+            body['Passphrase'] = passphrase
+        
+        for key, value in kwargs.items():
+            body[key] = value
+
+        body_json = json.dumps(body)
+        response = self._rest.POST(url=url, data=body_json)
+
+        return self.from_json(response.json())
+
+    def git_push(self, message: str, author: str, email: str, branch: str = None, new_branch: str = None, force: bool = None, username: str = None,
```

Comment:
`force` and `execute` are of type `bool` then the default value should be `False`